shared/actions/teams.js

```diff
@@ -918,6 +915,15 @@ const _onTabChange = (action: RouteTypes.SwitchTo) => {
   }
 }
 
+const _setChannelCreationError = (action: TeamsGen.SetChannelCreationErrorPayload) =>
```

Comment:
originally chat was handling this team action, but really teams should just handle this (or chat) but it shouldn't be across the two like before